redhat-developer/kam
# Executing kam bootstrap command

**What type of PR is this?**
/kind enhancement

**What does this PR do / why we need it**:
Adding e2e test coverage for kam bootstrap command
**Have you updated the necessary documentation?**


**Which issue(s) this PR fixes**:

Fixes #119

**How to test changes / Special notes to the reviewer**:
```make e2e``` test target should pass in CI

Test locally:

1. Login to the cluster as kubeadmin
2. The files ```kubeadmin-password``` and ```kubeconfig``` which contain cluster login details should be present in the auth directory and it should reside in the same directory as Makefile
2. Run ```make prepare-test-cluster```
3. Export these variables 
```
export SERVICE_REPO_URL=https://github.com/<your_github_org_name>/taxi
export GITOPS_REPO_URL=https://github.com/<your_github_org_name>/gitops.git
export IMAGE_REPO=quay.io/<your_quay_user_name>/taxi
export DOCKERCONFIGJSON_PATH=<Docker_credential_file_for_robot_account>
export GITHUB_TOKEN=<your_github_token>
```
4. Run ```make e2e```
